### PR TITLE
14092: Popper Component 

### PR DESCRIPTION
**Pull request by zoejab**

### CODE REVIEW

File: src/Popper/index.tsx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+      }
+    } else {
+      if (referenceElement.current !== null) {
+        referenceElement.current.focus()
+      }
+    }
+  }, [visible])
+
+  function handleClick() {
+    console.log('click')
```
Comment (craig1123) [inline src/Popper/index.tsx:88]: console

**Review by craig1123 — APPROVED** [review]

**Review by craig1123 — CHANGES_REQUESTED** [review]
Make sure you add "@popperjs/core" and "react-popper" to the externals in rollup.config.js

File: package.json [hunk 2]
```diff
@@ -26,10 +26,6 @@
     "watch": "rollup -c -w",
     "build:dist": "rimraf dist && node ./scripts/buildDist"
   },
-  "pre-commit": [
```
Comment (craig1123) [inline package.json:29]: you deleted my pre-commit :(

**Review by craig1123 — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (zoejab) [reply package.json:29]: thats weird. i definitely didn't manually do that. let me go get it back

Comment (zoejab) [reply package.json:29]: back and better than ever 😂

**Pull request merged**